src/Components/CreateImageWizard/formComponents/TargetEnvironment.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import useFieldApi from '@data-driven-forms/react-form-renderer/use-field-api';
+import { FormGroup, Tile } from '@patternfly/react-core';
+import './TargetEnvironment.scss';
+
+const TargetEnvironment = ({ label, isRequired, ...props }) => {
+    const { getState, change } = useFormApi();
+    const { input } = useFieldApi({ label, isRequired, ...props });
+    const [ environemt, setEnvironment ] = useState({
+        aws: false,
+        azure: false,
+        google: false,
+    });
+
+    useEffect(() => {
+        if (getState()?.values?.[input.name]) {
+            setEnvironment(getState().values[input.name]);
+        }
+    }, []);
+
+    return <FormGroup isRequired={ isRequired } label={ label } data-testid="target-select">
+        <div className="tiles">
+            <Tile
+                className="tile pf-u-mr-sm"
+                data-testid="upload-aws"
+                title="Amazon Web Services"
+                icon={ <img
+                    className='provider-icon'
+                    src={ '/apps/frontend-assets/partners-icons/aws.svg' } /> }
+                onClick={ () => setEnvironment((prevEnv) => {
+                    const newEnv = ({
+                        ...prevEnv,
+                        aws: !prevEnv.aws
+                    });
+                    change(input.name, newEnv);
+                    return newEnv;
+                }) }
+                isSelected={ environemt.aws }
+                isStacked
+                isDisplayLarge />
+            <Tile
+                className="tile"
```

Comment:
margin missing here

Reply:
I'll look into the css definition why it's not there.

Reply:
It's present in the other tiles, I think it might have happened when reordering